credentials/tls/certprovider/pemfile/watcher.go

```diff
@@ -191,23 +198,58 @@ func (w *watcher) updateRootDistributor() {
 		return
 	}
 
+	// If SPIFFEBundleMap is set, use it and DON'T use the RootFile, even if it
+	// fails
+	if w.opts.SPIFFEBundleMapFile != "" {
+		w.maybeUpdateSPIFFEBundleMap()
+	} else {
+		w.maybeUpdateRootFile()
+	}
+}
+
+func (w *watcher) maybeUpdateSPIFFEBundleMap() error {
+	// If the map file is unset, just return an error, don't create log spam.
+	if w.opts.SPIFFEBundleMapFile == "" {
```

Comment:
The only caller of this method is right above, `updateRootDistributor`, and that calls into here only if `w.opts.SPIFFEBundleMapFile != ""`. So, do we need this additional check here?